### DISCUSSION TITLE
exporting metrics sent 50%

### DISCUSSION THREAD

**Discussion by brl9** [netdata/netdata#13381] [Q&A] [answered]
hi2all.

i have the netdata parent node that is exported metrics to victoriametrics. and i have the "exporting" error - "exporting metrics sent 50%"

exporting.conf
```
[prometheus_remote_write:tokyo-victoriametrics1]
    enabled = yes
    destination = tokyo-victoriametrics1:8428
    remote write URL path = /api/v1/write
    update every = 1
    data source = as collected
    timeout ms = 20000
```
Victoriametrics has a default setup.

Some graphics:
<img width="1252" alt="Снимок экрана 2022-07-14 в 17 42 56" src="https://user-images.githubusercontent.com/74568202/179009664-617b3fd3-8956-41e8-a197-ad4ad22fafc3.png">

i can't understand. Is this a bug or a feature? :)

Netdata (Parent) - v1.35.1
VictroriaMetrics - v1.78.1

**Comment by ilyam8** [comment, marked as answer]
It was a bug. Fixed in #13435.
